src/extensions/scratch3_music/index.js

```
@@ -112,14 +112,19 @@ class Scratch3MusicBlocks {
     _storeSound (filePath, index, bufferArray) {
         const fullPath = `${filePath}.mp3`;
 
-        if (!assetData[fullPath]) return;
+        return this._fetchSounds()
+            .catch(() => ({}))
```

This does the same thing as the previous empty catch above?  Would you comment why that's ok to do here?

---

Yes. In case loading the sounds throws an error running the dynamic module or require.ensure is not available, this'll let the extension gracefully fail.